.eslintrc.json

```diff
@@ -5,9 +5,10 @@
     "node": true,
     "jquery": true,
     "amd": true,
-    "worker": true
+    "worker": true,
+    "jest/globals": true
   },
-  "extends": ["eslint:recommended", "plugin:prettier/recommended"],
+  "extends": ["eslint:recommended", "plugin:prettier/recommended", "airbnb-base", "plugin:jest/recommended"],
```

Comment:
What is the airbnb-base? Is it needed by jest?

Reply:
Its basically for es6 linting for the import export statements and other es6 syntax.

Reply:
Can we do the eslint thing with another PR? I don't strongly feel the need as I'm not seeing any real style difference after adding the plugin.